### PR TITLE
ISSUE-7 Provide better usage guidance in README

### PR DESCRIPTION
**Pull request by lewismc**
This issue is a work in progress to address #7 

@psundstrom can you suggest a efficient way which would allow the user to Select the entire folder containing the project, resources, etc. rather than having to manually import everything one artifact at a time.

Thank you

### CODE REVIEW

**Comment by psundstrom** [comment]
Hi,

If you would load the repo in the latest version of Impact (2023.2) all the packages and their dependencies would be loaded automatically when it is added to a workspace. This is defined in .impact/project.json.

/P

**Comment by lewismc** [comment]
@psundstrom thanks for the quick response 👍 

> If you would load the repo in the latest version of Impact (2023.2)...

Unfortunately we are using 2022.2 (Impact 1.8.0) right now. Is the guidance different for importing?

Thank you

**Comment by lewismc** [comment]
@psundstrom do you have any idea which versions of Modelon Impact are compatible with each branch? Thank you

**Comment by psundstrom** [comment]
> @psundstrom do you have any idea which versions of Modelon Impact are compatible with each branch? Thank you

Yes! 

vdl-4.1 - Impact 2022.2 (1.8.*)
vdl-4.2-upgrade - Impact 2023.1 (1.10.*)
vdl-4.3-upgrade/master - Impact 2023.2 (2.9.*)

**Comment by lewismc** [comment]
@psundstrom this PR is ready for review. Please take a look. Thank you 👍

File: README.md [hunk 1]
```diff
@@ -1,64 +1,46 @@
...
- - **DrivingUneven2** - Driving forward over uneven ground generated using the M2020 ZCAM -- Hogwallow Flats, Sol 461 3D model
+# Getting Started
 
-Acknowledgements
------------
-The visualization elements are extracted from https://github.com/nasa/NASA-3D-Resources/tree/master/3D%20Models/perseverance-GLB. This is also where the main geometrical parameters are gathered.
+Once logged into Modelon Impact Pro...
 
-The visualizer Perseverance/Resources/Spirit_landing_site.glb  is extracted from https://github.com/nasa/NASA-3D-Resources/tree/master/3D%20Models/Spirit%20Landing%20Site. This is also used to generate the VDL ground model data used in Perseverance.Experiments.DrivingUneven.
+1. Select **Repository Management** from the Apps menu.
```
Comment (psundstrom) [inline README.md:24]: This part is specific to an older version of Modelon Impact. We will update it later.

**Review by psundstrom — APPROVED** [review]
Approving to get the updates in. The instructions on loading the packages is specific to older versions of Impact. But since it's relevant at JPL I will approve it anyway.

**Pull request merged**